pkg/html/installhtml/install_by_operators.go

```diff
@@ -51,7 +51,10 @@ func summaryInstallRelatedTests(curr, prev sippyprocessingv1.TestReport, numDays
 	s := fmt.Sprintf(`
 	<table class="table">
 		<tr>
-			<th colspan=5 class="text-center"><a class="text-dark" title="Install related tests, sorted by passing rate.  The link will prepopulate a BZ template to be filled out and submitted to report a bug against the test." id="InstallRelatedTests" href="#InstallRelatedTests">Install Related Tests</a></th>
+			<th colspan=5 class="text-center">
+				<a class="text-dark" id="InstallRelatedTests" href="#InstallRelatedTests">Install Related Tests</a>
+				<i class="fa fa-info-circle" title="Install related tests, sorted by passing rate.  The link will prepopulate a BZ template to be filled out and submitted to report a bug against the test." id="UpgradeJobs" href="#UpgradeJobs">Upgrade Jobs</i>
```

Comment:
is this supposed to be install or upgrade?

Reply:
it's supposed to be neither. will remove